packages/elements/src/checkbox/index.ts

```diff
@@ -89,8 +89,8 @@ export class Checkbox extends ControlElement {
         this.indeterminate = false;
       }
 
-      this.ariaChecked = String(value);
-      void this.requestUpdate('checked', oldValue);
+      this.requestUpdate('checked', oldValue);
+      this.setAttribute('aria-checked', String(value));
```

Comment:
Why setAttribute after requestUpdate ? It should be ordered same as other properties that do setAttribute before requestUpdate.